.travis.yml

```diff
@@ -0,0 +1,39 @@
+language: cpp
+
+compiler:
+  - gcc
+
+before_install:
+  - sudo add-apt-repository ppa:ubuntu-toolchain-r/test -y
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq g++-4.9 doxygen python3-setuptools
+  - sudo easy_install3 gcovr cpp-coveralls
+  - wget https://googlemock.googlecode.com/files/gmock-1.7.0.zip -O ../gmock.zip
+  - unzip -qq ../gmock.zip -d ..
+  - mv ../gmock-1.7.0 ../gmock
+  - export CXX="g++-4.9" CC="gcc-4.9"
+
+script:
+  - mkdir build
+  - cd build
+  - mkdir installdir
+  - export BUILD_DIR=`pwd`
+  - export INST_DIR=`pwd`/installdir
+  - cmake -DCMAKE_INSTALL_PREFIX=$INST_DIR ..
+  - make
+
+after_success:
+  - make package
+  - make install
+  - cd $INST_DIR/share/doc/libproject
+  - tar -czf libproject-doc.tgz *
+  - mv libproject-doc.tgz $PREV_DIR
+  - cd $BUILD_DIR
+  - make run_tests
```

Comment:
*#bug*
tests running should be part of build
there no need to create package, when tests were failed
failed tests should fail build